### PR TITLE
Use Cloud Build for continuous integration

### LINKED ISSUE #26: Setup GitHub Actions CI
This repository is quickly going to need CI.

### PR DESCRIPTION
**Pull request by per-gron**
Fixes #26.

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+timeout: 3600s
+options:
+  machineType: 'N1_HIGHCPU_8'
+logsBucket: 'gs://bazel_rules_hdl'
+
```
[the remark this answers is not in the document]
Comment (per-gron) [reply cloudbuild.yaml:20]: Done.

**Comment by per-gron** [comment]
@QuantamHD @mithro Status update: I think by now the build actually works, but I have problems with timeouts. The build times out and fails after a bit over 40 minutes despite my attempts of updating the Cloud Build timeout to a larger value. I tried using the Buildbuddy.io Bazel cache but this also has problems, it times out and fails the build as it tries to upload some very large build artifacts.

**Comment by per-gron** [comment]
@QuantamHD This seems to finally work now! PTAL.

Things that I want to do in follow-up PRs: Build caching, building with GCC.

**Comment by QuantamHD** [comment]
This is awesome Per!!! Thanks for working on this it looks great.

**Pull request merged**